lambdas/service/app.py

```diff
@@ -877,7 +880,7 @@ def repository_seach_params_spec(index_name):
         filters_param_spec(facets),
         params.query(
             'size',
-            schema.optional(schema.with_default(10)),
+            schema.optional(schema.with_default(10, minimum=min_page_size, maximum=max_page_size)),
```

Comment:
It bugged me that we can only specify ranges for schemas that have a default so I generalized this a bit.